Source/Scene/Expression.js

```diff
@@ -13,14 +19,58 @@ define([
     function Expression(styleEngine, expression) {
         this._styleEngine = styleEngine;
 
-        console.log(jsep(expression));
+        var ast = jsep(expression);
+        console.log(ast);
+
+        this._runtimeAst = createRuntimeAst(this, ast);
+        console.log(this._runtimeAst);
     }
 
     defineProperties(Expression.prototype, {
     });
 
+    function Node(type, value, evaluate) {
+        this._type = type;
```

Comment:
Why explicitly store `type` now?  If we need it later, e.g., to expose as part of the API or for type conversion then let's add it then.